offload/liboffload/include/generated/OffloadAPI.h

```diff
@@ -477,7 +475,8 @@ OL_APIEXPORT ol_result_t OL_APICALL olMemFree(
 /// @brief Enqueue a memcpy operation.
 ///
 /// @details
-///    - For host pointers, use the device returned by olGetHostDevice
+///    - For host pointers, use the host device belonging to the
```

Comment:
This change isn't part of this MR, but I assume a previous update to the comments didn't get regen-ed.

Reply:
Yeah that's my bad. If we continue to check in the generated files it would be useful to have CI catch this by checking there's no diff after regenerating.